pallets/ddc-customers/src/lib.rs

```diff
@@ -223,12 +230,14 @@ pub mod pallet {
 		ArithmeticUnderflow,
 		// Transferring balance to pallet's vault has failed
 		TransferFailed,
+		/// Bucket is already removed
+		AlreadyRemoved,
 	}
 
 	#[pallet::genesis_config]
 	pub struct GenesisConfig<T: Config> {
 		pub feeder_account: Option<T::AccountId>,
-		pub buckets: Vec<(ClusterId, T::AccountId, BalanceOf<T>, bool)>,
+		pub buckets: Vec<(Bucket<T::AccountId>, BalanceOf<T>)>,
```

Comment:
change to Bucket<T> and then inside the Bucket definition, T: Config and owner_id to be T::AccountId

Reply:
Updated